src/components/accessories/admin/types/styles.scss

```diff
@@ -0,0 +1,27 @@
+@import "../../../../styles/variables";
+@import "../../../../../node_modules/susy/sass/susy";
```

Comment:
As [from offical doc](https://github.com/oddbird/susy): "Susy is Deprecated. It should no longer be used on new projects, and will no longer be receiving any updates." should we stop using it?

Yet, there are [50 other occurences of susy-media](https://github.com/search?q=repo%3Ainformatici%2Fopenhospital-ui+susy-media&type=code), so this is totally opened to discussion and if this is a technical change we want to do, let's handle it separately. wdyt?